### DISCUSSION TITLE
weather

### DISCUSSION THREAD

**Discussion by giscus** [siuze/NotionNext#9] [Announcements]
# weather

画君年少时, Notion, GitHub, 博客, 叶修齐

https://notion.siuze.top/weather

**Comment by htxzdfunny** [comment]
设备数据分析页面 404 了哦
![1716278198268.png](https://img2.imgtp.com/2024/05/21/h2bzN2Eo.png)

**Comment by siuze** [comment]
> 设备数据分析页面 404 了哦
> ![1716278198268.png](https://img2.imgtp.com/2024/05/21/h2bzN2Eo.png)

因为之前暂时关闭了自动更新数据的Action，有数据库安全问题不好解决。
